gulpfile.babel.js

```diff
@@ -23,6 +23,8 @@ var fs = require('fs-extra'),
 require('./gulp-tasks/tests')(gulp);
 require('./gulp-tasks/dist-' + process.platform)(gulp, reqs);
 
+var testAgent = 'RedHatDevelopmentSuiteInstaller/' + pjson.version + ' test/' + pjson.version;
```

Comment:
Everything seems to be fine, except this test agent string, it still would be a match for splunk query because regex we use is 
``` regexp
*RedHatDevelopmentSuiteInstaller*
```
Should be 'RedHatDevelopmentSuiteTestInstaller/' so our query would not count it.